open_academy/models/session.py

```diff
@@ -1,12 +1,21 @@
-from odoo import models, fields
+from odoo import models, fields, api
 
 
 class Session(models.Model):
     _name = 'session'
     name = fields.Char(required=True)
-    start_date = fields.Date()
+    start_date = fields.Date(default=fields.Date.today())
     duration = fields.Integer()
     number_of_seats = fields.Integer()
     instructor_id = fields.Many2one('res.partner')
     course_id = fields.Many2one('course', required=True)
     attendees_ids = fields.Many2many('res.partner', string="Attendees")
+    active = fields.Boolean(default=True)
+    taken_seats = fields.Integer(compute='_compute_percentage')
+    @api.depends('number_of_seats', 'attendees_ids')
+    def _compute_percentage(self):
```

Comment:
Check oca guidelines for models [here](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#models)

```suggestion
    def _compute_taken_seats(self):
```